environment_python_3_9.yml

```diff
@@ -62,6 +62,7 @@ dependencies:
     - pysqlite3==0.4.7
     - pyvo==1.4
     - redis
+    - selenium==4.8.0
```

Comment:
@bhilbert4 I've looked twice and didn't notice this package being imported in the changes you made? Do we need to include this dependency, am I not looking hard enough?

Reply:
Cool, I figured it was necessary but I didn't see the import declared in the code base anywhere so I just wanted to double check. Thanks!